tests/wpt/metadata/html/browsers/browsing-the-web/unloading-documents/navigation-within-beforeunload.html.ini

```diff
@@ -0,0 +1,4 @@
+[navigation-within-beforeunload.html]
```

Comment:
New failure, test hasn't been changed since 2015.

Reply:
Never mind, it had recent changes, I just didn't look at the right place.

Reply:
Never mind bis, the issue here is that 9467543c80c904915d7b6593a31d38f950a3c3d8 didn't get upstreamed correctly. Cc @jdm